### DISCUSSION TITLE
Custom notification text to identify download in database notifications

### DISCUSSION THREAD

**Discussion by howdu** [pxlrbt/filament-excel#206] [Ideas]
We need a way of identifying what these downloads are when using database notifications. 

<img width="448" alt="Screenshot 2024-11-22 at 12 18 18" src="https://github.com/user-attachments/assets/7a845393-c2eb-43dd-80f5-892060aa1682">

@pxlrbt would you accept PR for this? I think the easiest way would probably be adding to the exports cache, passing additional `$notificationTitle` into ExportFinishedEvent.

**Comment by pxlrbt** [comment]
Not 100% sure about the solution, but if you find a nice way to add additional information, I'm happy to merge.
